includes/admin/views/html-admin-page-entries-view.php

```diff
@@ -9,7 +9,7 @@
 
 $form_id    = isset( $_GET['form_id'] ) ? absint( $_GET['form_id'] ) : 0; // phpcs:ignore WordPress.Security.NonceVerification
 $entry_id   = isset( $_GET['view-entry'] ) ? absint( $_GET['view-entry'] ) : 0; // phpcs:ignore WordPress.Security.NonceVerification
-$entry      = evf_get_entry( $entry_id, true );
+$entry      = evf_get_entry( $entry_id );
```

Comment:
@roshansameer From my sense this has adverse affect on entries page, please have a look?

Reply:
I have tested it and I couldn't find anything wrong. In fact by adding that bool parameter the single entries meta boxes for the additional payment information was missing.